app/services/pickup_rate/__init__.py

```
@@ -0,0 +1,15 @@
+Monitors call pickup rates and sends Slack warning alerts when rates drop
+below configurable thresholds.
+"""
+
+from app.services.pickup_rate.monitor import PickupRateMonitor, pickup_rate_monitor
+from app.services.pickup_rate.task import initialize_pickup_rate_tasks
+
+__all__ = [
+    "PickupRateMonitor",
+    "pickup_rate_monitor",
+    "initialize_pickup_rate_tasks",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)